docs/guide/getting-started.md

```diff
@@ -1,22 +1,24 @@
-# Installation
+# Getting started
```

Comment:
The link in [guide/README.md file](https://github.com/chartjs/chartjs-plugin-deferred/blob/master/docs/guide/README.md#table-of-contents) needs to be renamed and updated. Same in the [root README.md](https://github.com/chartjs/chartjs-plugin-deferred#documentation).